src/main/java/educationalproject/programmingstuff/model/Item.java

```diff
@@ -1,10 +1,37 @@
 package educationalproject.programmingstuff.model;
 
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
 import java.math.BigDecimal;
 
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor //todo q: if it needed RequiredArgsConstructors for Entities?
+@Entity
+@Table(name="Items")
```

Comment:
Just BTW, SQL is case-insensitive, so it doesn't make sense to mix the cases. Also both cases are ok, choose the which one you like:)